airflow/providers/google/cloud/sensors/bigquery.py

```diff
@@ -64,25 +71,31 @@ def __init__(
         project_id: str,
         dataset_id: str,
         table_id: str,
-        bigquery_conn_id: str = 'google_cloud_default',
+        gcp_conn_id: str = 'google_cloud_default',
```

Comment:
Just an idea for further improvement - maybe it would be good to introduce constant for this value and use it in all cases?